packages/react-console/src/ConsoleSelector/ConsoleSelector.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Row, Col, Form, FormGroup, Dropdown, MenuItem, findChild } from 'patternfly-react';
+
+import { NONE_TYPE, SERIALCONSOLE_TYPE, VNCCONSOLE_TYPE } from './constants';
+
+class ConsoleSelector extends React.Component {
+  state = { type: NONE_TYPE };
```

Comment:
shouldn't this be in a constructor ? I get ```'state' is not defined. (no-undef)```

Reply:
This should be fine, it's a class property (more info i.e.: https://michalzalecki.com/react-components-and-class-properties)